src/org/labkey/targetedms/SkylineAuditLogManager.java

```diff
@@ -108,30 +134,38 @@ public boolean verifyPreRequisites() throws AuditLogException{
                 throw new AuditLogException("Current log integrity setting do not allow to upload a file without a valid document GUID. ");
         }
 
-        //retrieve number of documents with the same GUID
+        //retrieve count of documents with the same GUID
         TableInfo runsTbl = TargetedMSManager.getTableInfoRuns();
-        SimpleFilter guidFilter = new SimpleFilter(FieldKey.fromParts("documentGuid"), _documentGUID.toString());
-
-        SQLFragment query = new SQLFragment("SELECT count(*) docCount FROM ").append(runsTbl.getSelectName()).append(" ");
-        query.append(guidFilter.getSQLFragment(TargetedMSManager.getSqlDialect()));
-
-        BaseSelector.ResultSetHandler<Integer> resultSetHandler = (rs, conn) -> {
-            if(rs.next()){
-                return rs.getInt("docCount");
-            }
-            else
-                return 0;
-        };
-        Integer docCount = new SqlExecutor(runsTbl.getSchema()).executeWithResults(query, resultSetHandler);
-        _logTree = buildLogTree();
+        SimpleFilter.AndClause and = new SimpleFilter.AndClause();
```

Comment:
This should work fine, but I don't think you need the AndClause. You can just append all of these directly to docFilter, since all of the clauses get AND'd together.

AndClause is useful when you're composing a more complex set of criteria into a single clause, often in conjunction with OrClause.